### ISSUE TITLE
Text field limit translation not working

### ISSUE THREAD

**Issue by mini71** [verbb/formie#1053]
### Describe the bug

I added a translation of '{num} characters left' to a formie.php translation file but there is no effect at the rendered form.
Other translations are working as expected. 'window.FormieTranslations' does contain the right translation.

### Steps to reproduce

Enable 'Limit Field Content'
Add a custom translation file @translations/de/formie.php
Add a translation for '{num} characters left'

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

4.2.1.1

### Plugin version

2.0.7

### Multi-site?

Yes

**Comment by engram-design** [comment]
Good point, this was an undocumented change in 1.6.6/2.0.5 where the translation string was changed to `{startTag}{num}{endTag} characters left` so that we could style the number differently after it reaches zero. This is because the limits are now "soft" (Twitter-style) in that it allows for overtyping.

To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.7",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.8](https://github.com/verbb/formie/releases/tag/2.0.8)
